mergin/test/test_client.py

```diff
@@ -2742,3 +2742,58 @@ def test_workspace_requests(mc2: MerginClient):
     assert service["plan"]["product_id"] == None
     assert service["plan"]["type"] == "custom"
     assert service["subscription"] == None
+
+
+def test_access_management(mc: MerginClient):
```

Comment:
could you also add tests for some failures, e.g. permissions issues, duplicate calls, limits?